src/routes/api/profile/education/languages/update/+server.ts

```diff
@@ -0,0 +1,25 @@
+import { BACKEND_BASE_URL } from '$env/static/private'
+import { json } from '@sveltejs/kit'
+
+export interface LanguageUpdatePayload {
+  languageId: number
+  proficientLevel: string
+}
+
+export async function POST({ request, fetch }: { request: Request; fetch: typeof window.fetch }) {
```

Comment:
```suggestion
export async function PUT({ request, fetch }: { request: Request; fetch: typeof window.fetch }) {
```
Como actualiza algo existente, debiera ser un PUT en vez de un POST. Y sí, SvelteKit sí lo soporta https://learn.svelte.dev/tutorial/other-handlers